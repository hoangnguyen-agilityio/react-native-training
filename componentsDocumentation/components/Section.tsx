import React from 'react';
import {StyleSheet, View} from 'react-native';

const Section = (props: any) => <View style={styles.section} {...props} />;

const styles = StyleSheet.create({
  section: {
    marginTop: 24,
    paddingVertical: 16,
    borderBottomColor: '#737373',
    borderBottomWidth: 2,
  },
});

export default Section;
