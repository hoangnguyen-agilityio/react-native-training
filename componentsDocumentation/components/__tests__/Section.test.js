import React from 'react';
import renderer from 'react-test-renderer';
import Section from '../Section';

test('renders correctly', () => {
  const tree = renderer.create(<Section />).toJSON();
  expect(tree).toMatchSnapshot();
});