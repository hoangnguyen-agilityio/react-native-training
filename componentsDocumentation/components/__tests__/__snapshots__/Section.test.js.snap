// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderBottomColor": "#737373",
      "borderBottomWidth": 2,
      "marginTop": 24,
      "paddingVertical": 16,
    }
  }
/>
`;
